tools/export/nb/__init__.py

```diff
@@ -207,11 +211,19 @@ def create_jinja_ctx(self):
         sources = [self.filter_dot(field) for field in sources]
         include_paths = [self.filter_dot(field) for field in self.resources.inc_dirs]
 
+        sys_libs = [self.prepare_sys_lib(lib) for lib
+                    in self.toolchain.sys_libs]
+        preproc = " ".join([part for part
```

Comment:
You don't need this list comprehension. Could you remove it?

Reply:
Sorry, I don't understand. Could you please more specific?

Reply:
```
        preproc = " ".join([part for part
                            in ([basename(self.toolchain.preproc[0])] +
                                self.toolchain.preproc[1:] +
                                self.toolchain.ld[1:])])
```

==

```
        preproc = " ".join([basename(self.toolchain.preproc[0])] +
                           self.toolchain.preproc[1:] +
                           self.toolchain.ld[1:])
```

Reply:
Changed!